### PR TITLE
Update next with master

### PR DESCRIPTION
**Pull request by zFernand0**
This PR should update system tests since it includes changes in #1116

### CODE REVIEW

**Comment by zFernand0** [comment]
Before:
```
Test Suites: 23 failed, 102 passed, 125 total
Tests:       55 failed, 4 skipped, 813 passed, 872 total
Snapshots:   1 failed, 95 passed, 96 total
```
After:
```
Test Suites: 13 failed, 112 passed, 125 total
Tests:       24 failed, 4 skipped, 845 passed, 873 total
Snapshots:   95 passed, 95 total
```

**Review by gejohnston — DISMISSED** [review]

**Review by awharn — DISMISSED** [review]

File: packages/core/CHANGELOG.md [hunk 1]
```diff
@@ -14,10 +14,6 @@ All notable changes to the Zowe core SDK package will be documented in this file
 
 - BugFix: Updated dependencies to resolve problems with the ansi-regex package
 
-## `6.33.1`
-
-- Migrated from TSLint (now deprecated) to ESLint for static code analysis.
-
```
Comment (t1m0thyj) [inline packages/core/CHANGELOG.md:20]: Why was this removed? Same question applies to the changelogs below.

Comment (zFernand0) [reply packages/core/CHANGELOG.md:20]: IIRC we made the decision to not document dev-only releases in the changelog.

Comment (t1m0thyj) [reply packages/core/CHANGELOG.md:20]: Generally yes, but IIRC we published the patch release 6.33.1 for no other reason than migrating from TSLint to ESLint, and agreed to add the line to the changelog then, so I'd be hesitant to remove it now.

Comment (zFernand0) [reply packages/core/CHANGELOG.md:20]: Makes sense 👍 
I'll ad it back : )

File: packages/zosuss/CHANGELOG.md [hunk 2]
```diff
@@ -16,13 +16,13 @@ All notable changes to the Zowe z/OS USS SDK package will be documented in this
 
 - Documentation: Update README with information about the new optional native modules in ssh2 dependency
 
-## `6.33.2`
+## `6.33.1`
```
Comment (awharn) [inline packages/zosuss/CHANGELOG.md:19]: Why is this backwards?

Comment (zFernand0) [reply packages/zosuss/CHANGELOG.md]: oops

**Review by t1m0thyj — APPROVED** [review]
LGTM! Thanks for adding the "Development" tag to the changelog 🙂
